Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -742,7 +748,7 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     for (auto& pattern : matchPatterns) {
         bool matchFound = false;
         for (auto& grantedPattern : currentPermissionMatchPatterns()) {
-            if (pattern->matchesPattern(grantedPattern, { WebExtensionMatchPattern::Options::IgnorePaths })) {
+            if (grantedPattern->matchesPattern(pattern, { WebExtensionMatchPattern::Options::IgnorePaths })) {
```

Comment:
I'm not 100% convinced switching this is correct. will confirm in the morning when I re-test this.